torchbenchmark/score/compute_score_v0.py

```diff
@@ -0,0 +1,135 @@
+"""
+Compute the benchmark score given a frozen score configuration and current benchmark data.
+"""
+import argparse
+import json
+import math
+import sys
+import os
+import re
+import yaml
+import importlib
+
+from tabulate import tabulate
+from pathlib import Path
+from collections import defaultdict
+
+from .generate_score_config import generate_bench_cfg
```

Comment:
Out of curiosity, how come we use relative imports?

Reply:
Because in V0 the code to generate the norm yaml file is in generate_score_cfg.py, so we use relative import to import that. This is for compatibility purpose.